vcxagency-node/src/api/middleware.js

```diff
@@ -57,6 +57,21 @@ module.exports.asyncHandler = function asyncHandler (fn) {
   }
 }
 
+module.exports.buildDenyQueryStringsMiddleware = function buildDenyQueryStringsMiddleware (allowedQueryKey) {
```

Comment:
Maybe a better solution would be to take an array of query key exceptions and test that the actual query is a subset of the query key exceptions (so that it is easier to add new exceptions in the future)?